### PR TITLE
Additions to the TOC for suggested topics

### PR DESCRIPTION
**Pull request by JonasEconomist**
I've added some other topics. If these sound good to you all, I'll start creating pages for them. I'll also review what is already there next.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,5 +24,17 @@ As a consequence of the above considerations, a set of well defined best practic
 
 ### * [Comments](COMMENTS.md)
 ### * PR Process [Discuss if we can reuse this ](https://github.com/EconomistDigitalSolutions/fe-blogs/blob/master/docs/CODE_REVIEW.md)
+### * Git Workflow: [We use this, it's good for the goal of being open source someday](https://www.atlassian.com/git/tutorials/comparing-workflows/forking-workflow)
```
Comment (sandeepasrani) [inline README.md]: https://confluence.atlassian.com/bitbucket/branch-or-fork-your-repository-221450630.html
Personall prefer the Branching approach.

Comment (umbobabo) [reply README.md]: @sandeepasrani can you motivate it, pls? Otherwise is complex to take a decision or create consensus.

Comment (sandeepasrani) [reply README.md]: I agree I will do that just been busy with Auth0 and all the context switching. And I am now part of CASE. I call myself where all the shit is thrown at. Anyway this is something I need to push. So will do that.

Apologize for the lazing around ? :)

Comment (umbobabo) [reply README.md]: Np @sandeepasrani . Take your time for this stuff 👍

**Pull request merged**